packages/babel-core/src/config/validation/options.js

```diff
@@ -187,6 +191,7 @@ export type ValidatedOptions = {
   inputSourceMap?: RootInputSourceMapOption,
   envName?: string,
   caller?: CallerMetadata,
+  showConfig?: boolean,
```

Comment:
Should the changes in this file be reverted because of https://github.com/babel/babel/pull/11588/commits/a4d31fa32ab4ded7e314529260357f0db6402d14?

Reply:
Fixed(up) in cb3b76bba74e131c50fc4d2c17325a80132a15aa.